src/web/page/templates.rs

```diff
@@ -65,7 +64,9 @@ impl TemplateData {
 
             while rx.recv().is_ok() {
                 if let Err(err) = reload(&template_data, &pool) {
-                    log::error!("failed to reload templates: {:?}", err);
+                    log::error!("Failed to reload templates:\n{}", err);
+                } else {
+                    log::info!("Reloaded templates");
```

Comment:
Logging should be lower-case so it doesn't look odd after another prefixed message: https://rust-lang.github.io/api-guidelines/interoperability.html#error-types-are-meaningful-and-well-behaved-c-good-err
```suggestion
                    log::error!("failed to reload templates: {}", err);
                } else {
                    log::info!("reloaded templates");
```